lang/user/id.xml

```diff
@@ -227,15 +223,14 @@ Kode kesalahan: {}</an_error_occurred>
 		<one_minute_ago>1 Minit Yg Lalu</one_minute_ago>
```

Comment:
"1 Menit yang lalu"

Reply:
Can't it be long bro, please bro don't keep protesting or there won't be any updates